sdk/core/azure_core_amqp/Cargo.toml

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Microsoft Corp. All Rights Reserved.
+
+# AMQP Stack for consumption by packages in the Azure SDK.
+[package]
+name = "azure_core_amqp"
+version = "0.19.0"
+description = "Rust client Azure SDK"
+readme = "README.md"
+authors = ["Microsoft Corp."]
+license = "MIT"
+repository = "https://github.com/azure/azure-sdk-for-rust"
+homepage = "https://github.com/azure/azure-sdk-for-rust"
```

Comment:
Nit: we should send them to this crates' subdir.

Some other info in here will be changed, but not necessary now.